camel/societies/role_playing.py

```diff
@@ -90,60 +94,124 @@ def __init__(
         extend_task_specify_meta_dict: Optional[Dict] = None,
         output_language: str = None,
     ) -> None:
+        self.assistant_role_name = assistant_role_name
```

Comment:
Why do we need to save so many variables into fields? This many fields in a class point as poor cohesion.

Reply:
Will remove them.